lib/locale/pl/strings.php

```diff
@@ -24,6 +24,8 @@
  *  $Id$
  */
 
+$_LANG['Sent ticket to verificator'] = 'Zgłoszenie wysłane do weryfikatora';
+$_LANG['Send ticket to verificator'] = 'Przekaż zgłoszenie do weryfikatora';
```

Comment:
Tu jest w znaczeniu przekaż czy wyślij?
Bo przekaż to po angielsku raczej "transfer".

Reply:
Tak w znaczeniu Wyślij. Chciałem te dwa zdania optycznie bardziej odróżnić.

Reply:
Zaraz poprawie inaczej.